src/main/java/cz/cvut/kbss/termit/event/VocabularyRemovalEvent.java

```diff
@@ -0,0 +1,23 @@
+package cz.cvut.kbss.termit.event;
+
+import cz.cvut.kbss.termit.model.Vocabulary;
+import org.springframework.context.ApplicationEvent;
+
+import java.net.URI;
+
+/**
+ * Indicates that a Vocabulary will be removed
+ * <p>
+ * Fired in {@link cz.cvut.kbss.termit.persistence.dao.VocabularyDao#remove(Vocabulary)} right before the vocabulary removal.
```

Comment:
The event should not state by which component it is fired; it is not its job to know such thing. Also, if the event is fired before vocabulary removal (and the **before** is important), its name should indicate that.